content/source/docs/enterprise/install/active-active.html.md

```diff
@@ -22,34 +22,32 @@ However, the benefits that come with the Active/Active architecture need to be w
+The application itself must be using [External Services](https://www.terraform.io/docs/enterprise/before-installing/index.html#operational-mode-decision) mode to connect to an external PostgreSQL database and object storage. 
 
-All admin and application configuration must be automated via your settings files, i.e. it cannot have been configured via the Admin UI. Specifically, you should be using the following configuration files:
+All admin and application configuration must be automated via your settings files and current with running configuration, i.e. it cannot have been altered via the Admin UI and not synced to the file. Specifically, you should be using the following configuration files:
 
 
+*   `/etc/replicated.conf`    - contains the configuration for the Replicated installer
+*   `/etc/ptfe-settings.json` - contains the configuration for the TFE application 
 
-*   **/etc/replicated.conf** which houses the configuration for the Replicated installer
-*   **/etc/ptfe-settings.json** which controls the configuration of TFE itself
+-> **Note**: The location for the latter is controlled by the "ImportSettingsFrom" setting in `/etc/replicated.conf` and is sometimes named `settings.json` or `replicated-tfe.conf`
 
--> **Note**: The location for the latter is controlled by the "ImportSettingsFrom" setting in `/etc/replicated.conf`
-
-The requirement for automation is two-fold. First, the nodes need to be able to spin up and down without human intervention. More importantly though, you will be disabling the Admin UI altogether later in this guide. **The Admin UI does not function correctly when running multiple nodes and must not be used** after Step 3 below.
+The requirement for automation is two-fold. First, the nodes need to be able to spin up and down without human intervention. More importantly though, you will need to ensure configuration is managed in this way going forward as the Admin UI will be disabled. **The Admin UI does not function correctly when running multiple nodes and must not be used**.
 
 
 ### Step 1: Prepare to Externalize Redis
 
 
 #### Prepare Network
 
-There are two new considerations when thinking about ingress and egress:
-
+There are new access requirements involving ingress and egress:
 
 
-*   **Port 6379** (or the port the external Redis will be configured to use below) must be open between the instances and the Redis service
-*   **Port 8201** must be open between the instances to allow Vault to run in [High Availability](https://www.vaultproject.io/docs/internals/high-availability) mode
-*   **Port 8800** should be closed; since the admin dashboard is no longer available when running multiple nodes
+*   **Port 6379** (or the port the external Redis will be configured to use) must be open between the node instances and the Redis service
+*   **Port 8201** must be open between the node instances to allow Vault to run in [High Availability](https://www.vaultproject.io/docs/internals/high-availability) mode
```

Comment:
```suggestion
*   **Port 6379** (or the port the external Redis will be configured to use) must be open between the nodes and the Redis service
*   **Port 8201** must be open between the nodes to allow Vault to run in [High Availability](https://www.vaultproject.io/docs/internals/high-availability) mode
```

Reply:
I had actually standardized on node instance, but missed some. I debated just node, but wasn't sure we wanted to depict too much of a true "cluster", so went node instance. I think just node cleaner and since you prepared the commits already, will accept all those revisions.